packages/design-system/src/Avatar/Avatar.styles.tsx

```diff
@@ -6,6 +6,14 @@ import {
 import { AvatarSize } from "./Avatar.types";
 
 const Variables = css`
+  --ads-v2-colors-content-avatar-surface-default-bg: var(
```

Comment:
Can we add a blurb on why Avatar is not following the response category token pattern?

Reply:
This will be added in documentation once @vasanthappsmith  provides it.

Reply:
I think you can also write it - a one line rationale on why we're making this decision is all.